testing/e2e/selenium-webdriver-controller.js

```diff
@@ -210,7 +224,54 @@ export class SeleniumWebDriverController {
       width,
       height,
     } = rect;
-    await this.driver.manage().window().setRect({width, height});
+
+    // Calculate the window borders, so we can set the correct size to get the
+    // desired content size.
+    const results = await Promise.all([
+      this.driver.manage().window().getRect(),
+      this.driver.findElement(By.tagName('html')),
+    ]);
+    // No Array destructuring allowed?
```

Comment:
That was introduced in https://github.com/ampproject/amphtml/pull/4217 but I haven't found a rationale behind it. I can speculate that in most cases it's too terse, but it would be really nice for getting results from `Promise.all`